torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -119,51 +119,57 @@ def get_num_frames(inpt: features.VideoTypeJIT) -> int:
         raise TypeError(f"The video should be a Tensor. Got {type(inpt)}")
 
 
-def _xywh_to_xyxy(xywh: torch.Tensor) -> torch.Tensor:
-    xyxy = xywh.clone()
+def _xywh_to_xyxy(xywh: torch.Tensor, inplace: bool) -> torch.Tensor:
+    xyxy = xywh if inplace else xywh.clone()
     xyxy[..., 2:] += xyxy[..., :2]
     return xyxy
 
 
-def _xyxy_to_xywh(xyxy: torch.Tensor) -> torch.Tensor:
-    xywh = xyxy.clone()
+def _xyxy_to_xywh(xyxy: torch.Tensor, inplace: bool) -> torch.Tensor:
+    xywh = xyxy if inplace else xyxy.clone()
     xywh[..., 2:] -= xywh[..., :2]
     return xywh
 
 
-def _cxcywh_to_xyxy(cxcywh: torch.Tensor) -> torch.Tensor:
+def _cxcywh_to_xyxy(cxcywh: torch.Tensor, inplace: bool) -> torch.Tensor:
     cx, cy, w, h = torch.unbind(cxcywh, dim=-1)
     x1 = cx - 0.5 * w
     y1 = cy - 0.5 * h
     x2 = cx + 0.5 * w
     y2 = cy + 0.5 * h
-    return torch.stack((x1, y1, x2, y2), dim=-1).to(cxcywh.dtype)
+    xyxy = torch.stack((x1, y1, x2, y2), dim=-1).to(cxcywh.dtype)
+    if inplace:
+        return cxcywh.fill_(xyxy)
+    return xyxy
```

Comment:
That is faithful to the `inplace` parameter, but the whole point of it is to make the call faster. This will only make it slower since we already have a new storage. If we currently don't have a faster inplace version, should we maybe not do that here?

Really depends on how we interpret `inplace`.

1. Operation might be performed inplace for speed
2. Operation will be performed inplace

I'm in favor of 1. here, but I think both are valid.

Reply:
It's marked as draft for a reason. I'm not done yet 😄

Reply:
I found a solution that works without the issues described here. Let me know what you think.